src/otaclient/ota_core.py

```diff
@@ -245,26 +244,6 @@ def __init__(
         )
         self._downloader_mapper: dict[int, Downloader] = {}
 
-    def _calculate_delta(
-        self,
-        standby_slot_creator: StandbySlotCreatorProtocol,
-    ) -> DeltaBundle:
-        logger.info("start to calculate and prepare delta...")
-        delta_bundle = standby_slot_creator.calculate_and_prepare_delta()
-
-        # update dynamic information
-        self._status_report_queue.put_nowait(
-            StatusReport(
-                payload=SetUpdateMetaReport(
-                    total_download_files_num=len(delta_bundle.download_list),
-                    total_download_files_size=delta_bundle.total_download_files_size,
-                    total_remove_files_num=len(delta_bundle.rm_delta),
-                ),
-                session_id=self.session_id,
-            )
-        )
-        return delta_bundle
-
```

Comment:
this function is moved to https://github.com/tier4/ota-client/pull/486/files#diff-210b85bcdeb75a7247725e9775711ab1d55faef4dda8386641bf1e9f6ee86cbfR523-R536